onnxruntime/core/session/inference_session.cc

```diff
@@ -946,6 +971,46 @@ class InferenceSession::Impl {
     session_state_.SetLogger(*session_logger_);
   }
 
+  // Registers all the predefined transformers with transformer manager
+  void AddPredefinedTransformers(GraphTransformerManager& transformer_manager, const uint32_t& enabled_levels, const std::vector<std::string>& custom_list) {
+    if ((enabled_levels & TransformerLevel::Optional_L1) || !custom_list.empty()) {
```

Comment:
Limit lines to 120 chars. [https://marketplace.visualstudio.com/items?itemName=PaulHarrington.EditorGuidelines](https://marketplace.visualstudio.com/items?itemName=PaulHarrington.EditorGuidelines) is very helpful for this.